sirepo/package_data/static/html/index.html

```diff
@@ -83,24 +83,24 @@
     <script src="/static/js/ext/jquery-2.2.4.min.js{{ source_cache_key }}"></script>
```

Comment:
maybe this whole thing should be a for loop in jinja with the modules coming from python schema: staticModules. That way we can use them programmatically elsewhere:

```txt
{% for m  in static_js_files %}
<script src="{{ }}"></script>
{% endfor %}
```

The creation of static_js_files would be inside the python code schema. The load order could be there, too. This would allow more flexibility and could reuse "static" from uri_router or whatever.